convert-policy.py

```diff
@@ -55,21 +54,19 @@ def rewrite_defs(policy_markdown):
     return re.sub(r'<a id=([^>]*)>[*][*]([^*]*)[*][*]</a>', '<dfn id=\\1>\\2</dfn>', policy_markdown)
 
 
-def fix_nested_lists(policy_markdown):
-    return re.sub(r'^   1[.]', '    1.', policy_markdown, flags=re.MULTILINE)
-
-
-def avoid_link_false_positives(policy_markdown):
-    return re.sub(r'[]] [(]', '] \\(', policy_markdown)
-
-
 def preprocess_markdown(policy_markdown, mapping_pairs):
     result = lower_headers(policy_markdown)
     result = apply_link_mapping(result, mapping_pairs)
-    result = add_header_anchors(result)
     result = rewrite_defs(result)
-    result = fix_nested_lists(result)
-    result = avoid_link_false_positives(result)
+
+    return result
+
+
+def postprocess_html(policy_html, template, title):
+    result = policy_html.replace("&quot;", "\"")
```

Comment:
This seems dangerous as a blanket transformation. Should we just let Commonmark do its thing?

Reply:
No, that will create really messy IDs. This is needed for a minimal diff with what we have today. I guess the main future problem would be when setting the title attribute through Markdown?

Reply:
Yeah, or any raw HTML uses.

I'm OK with a non-minimal diff, but if this will affect ID generation that's not great. Maybe the ID generation code should do this transformation itself before generating the ID?

Reply:
Could you review https://github.com/whatwg/whatwg.org/pull/294/commits/f51ca893eac8620347a2b0f7147eb36cf1f59e11 to ensure that's all that's changed? I guess another thing to consider is using better quotation marks in the source material, but that might well require changes to header to ID conversion as well.

Reply:
Reviewed! Looks good to me too; thanks for going the extra step.